src/static/partials/add-workshop.html

```diff
@@ -34,27 +39,33 @@
       <label class="col-md-4 control-label">Location of Workshop *</label>  
       <div class="col-md-4 inputGroupContainer">
 	<div class="input-group">
-	  <input type="text" name="location" ng-model="location" ng-pattern="/^[a-zA-Z\s,]*$/"  class="form-control"
-		 placeholder="College Name, City Name, State Name, Country Name" required>
+	  <input type="text" ng-mouseover="targetWorkshopLocation('over')" ng-mouseleave="targetWorkshopLocation('unover')" name="expts" min="0" ng-model="expts"  class="form-control"
```

Comment:
name and ng-model should be name="location" and ng-model="location" instead of
name="expts"
ng-model="expts".
The above changes will break the functionality of adding workshop

Reply:
Fixed.
https://github.com/vlead/outreach-portal/commit/2b00dbfc509e84680e8829026d7bfcb720903342